pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/CompatibilityOpsRunner.java

```diff
@@ -36,8 +36,7 @@ private CompatibilityOpsRunner(String configFileName) {
     _configFileName = configFileName;
   }
 
-  private boolean runOps()
-      throws IOException, JsonParseException, JsonMappingException {
+  private boolean runOps() throws IOException, JsonParseException, JsonMappingException, InterruptedException {
```

Comment:
(nit) Please set up the Pinot Code Style following the instructions [here](https://docs.pinot.apache.org/developers/developers-and-contributors/code-setup) and reformat the changes

Reply:
+1